internal/db/q.go

```diff
@@ -25,7 +25,10 @@ type FieldMatcher struct {
 
 // could be faster probably
 func (fm FieldMatcher) Match(st Record) (bool, error) {
-	candidate := st.Get(fm.field)
+	candidate, err := st.Get(fm.field)
```

Comment:
damn, checking err here is probably actually pretty expensive. Maybe we should give record a MustGet method that just panics on error.